pkg/cluster/nodes/node.go

```diff
@@ -167,7 +167,17 @@ func (n *Node) LoadImages() {
 		// use xargs to load images in parallel
 		`find /kind/images -name *.tar -print0 | xargs -0 -n 1 -P $(nproc) docker load -i`,
 	).Run(); err != nil {
-		log.Warningf("Failed to preload docker images: %v", err)
+		log.Warningf("Failed to preload images into docker: %v", err)
+		return
+	}
+
+	// load images cached on the node into containerd
+	if err := n.Command(
+		"/bin/bash", "-c",
+		// use xargs to load images in parallel
+		`docker images --format "{{.Repository}}:{{.Tag}}" | while read -r line; do echo "$line" && docker save "$line" > tmp.tar && ctr --namespace=k8s.io images import --snapshotter=native tmp.tar && rm tmp.tar; done`,
```

Comment:
i don't understand very well how having the images loaded in both docker and containerd affects the node in terms of e.g. memory footprint. maybe there is some magic behind the scenes to minimize this, but i wonder if we have a case where we now have the images loaded twice?

do we need a knob in kind to do that.
the kubeadm config socket is one way, but for that:
- kind needs to process the kubeadm config to check if the socket is different from the default
- we need to propagate this "knob" here so that the Node object and LoadImages know about it.

Reply:
we won't leave it this way, but this is a reasonable enough answer until we add the more complex answer I think

Reply:
first kind will need to know if the image supports this, to do that we'll start detecting metadata from the image (labels?) to identify features

then kind can simply skip the docker load when it detects that it doesn't need to use dockershim
the `kind load` commands will also need to support this, and the proxy configuration will need to handle containerd.

Reply:
FTR this increases disk footprint at runtime, but not so much cpu / memory.